llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -14328,7 +14328,7 @@ SDValue AArch64TargetLowering::LowerVectorSRA_SRL_SHL(SDValue Op,
       unsigned Opc =
           (Op.getOpcode() == ISD::SRA) ? AArch64ISD::VASHR : AArch64ISD::VLSHR;
       return DAG.getNode(Opc, DL, VT, Op.getOperand(0),
-                         DAG.getConstant(Cnt, DL, MVT::i32));
```

Comment:
unrelated?

Reply:
This one is part of another pull request. The dependencies are like this:

#92922 <--- #92528 ---> #92529